lib/config/configuration.go

```diff
@@ -2267,3 +2275,30 @@ func applyTokenConfig(fc *FileConfig, cfg *servicecfg.Config) error {
 
 	return nil
 }
+
+func applyOktaConfig(fc *FileConfig, cfg *servicecfg.Config) error {
+	if fc.Okta.APIEndpoint == "" {
+		return trace.BadParameter("okta_service is enabled but no api_endpoint is specified")
+	}
+	if fc.Okta.APITokenPath == "" {
+		return trace.BadParameter("okta_service is enabled but no api_token_path is specified")
+	}
+
+	// Make sure the URL is valid
+	url, err := url.Parse(fc.Okta.APIEndpoint)
```

Comment:
`url.Parse` can return nil if no Host/schema is provided.
It's probably better to check them as well if required